docs/notes/TypecheckingCosts/Costs.md

```diff
@@ -0,0 +1,7 @@
+Typechecking costs are relatively simple: we say that each reduction step
```

Comment:
We've been putting notes in the source code following roughly the GHC convention. Have a look for `Note` comments elsewhere. I much prefer having them in the source since they're more discoverable.

Reply:
This file is still committed - did you mean to delete it?